File: Backend (Springboot)/Annapurna/src/main/java/com/annapurna/dto/OrderDTO.java
package com.annapurna.dto;

import lombok.Data;

@Data
public class OrderDTO {
	private Long tiffinId; 
    private Long orderId;
    private String tiffinName;
    private int quantity;
    private double totalPrice;
    private String status;

    private String customerName;
    private String customerEmail;
    private String customerMobileNumber;
}

File: Backend (DNET)/Annapurna/DTOs/OrderDTO.cs
namespace AnnapurnaAPI.DTOs
{
    public class OrderDTO
    {
        public long TiffinId { get; set; }
        public long OrderId { get; set; }
        public string TiffinName { get; set; } = string.Empty;
        public int Quantity { get; set; }
        public double TotalPrice { get; set; }
        public string Status { get; set; } = string.Empty;
        public string CustomerName { get; set; } = string.Empty;
        public string CustomerEmail { get; set; } = string.Empty;
        public string CustomerMobileNumber { get; set; } = string.Empty;
    }
}
